.snippets/code/eth-api/libraries/viem/deploy.ts

```diff
@@ -0,0 +1,47 @@
+// 1. Update imports
+import { createPublicClient, createWalletClient, http } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { moonbaseAlpha } from 'viem/chains';
+import contractFile from './compile';
+
+// 2. Create a wallet client for writing chain data
```

Comment:
Maybe mention here in a comment that the private key needs to be prepended with the `0x`. If you omit the 0x, as MetaMask does when copying private keys, you'll get a nondescript error.